python/ql/src/analysis/AlertSuppression.ql

```diff
@@ -72,7 +72,7 @@ class LgtmSuppressionComment extends LineSuppressionComment {
  * A noqa suppression comment. Both pylint and pyflakes respect this, so lgtm ought to too.
  */
 class NoqaSuppressionComment extends LineSuppressionComment {
-  NoqaSuppressionComment() { this.getContents().toLowerCase().regexpMatch("\\s*noqa\\s*") }
+  NoqaSuppressionComment() { this.getContents().toLowerCase().regexpMatch("\\s*noqa([^:].*)?") }
```

Comment:
Note that `noqa : F401` (with a space before the colon) appears to be legal, in which case I would recommend something like this:
```
"\\s*noqa\\s*([^:].*)?"
```
instead of:
```
"\\s*noqa([^:].*)?"
```